src/main/java/AuthLanding.java

```diff
@@ -0,0 +1,41 @@
+// public class AuthLanding implements AuthInterface {
```

Comment:
why is this commented out?

Reply:
It was functioning properly so I just commented out until i figured it out to make it work properly